src/JsFunctionsScanner.php

```diff
@@ -106,39 +106,23 @@ public function saveGettextFunctions( Translations $translations, array $options
 			switch ( $functions[ $callee->getName() ] ) {
 				case 'text_domain':
 				case 'gettext':
-					if ( ! isset( $args[1] ) ) {
-						break;
-					}
-
-					list( $original, $domain ) = $args;
+					list( $original, $domain ) = array_pad( $args, 2, null );
```

Comment:
Ah, padding with `null` to avoid PHP errors when you're short on arguments. Nice one!